ppr-ui/src/enums/registrationTypes.ts

```diff
@@ -194,17 +194,17 @@ export enum APIMhrDescriptionTypes {
   RESIDENTIAL_EXEMPTION = 'RESIDENTIAL EXEMPTION',
   NON_RESIDENTIAL_EXEMPTION = 'NON-RESIDENTIAL EXEMPTION',
 
-  ABAN = 'ABAN',
-  BANK = 'BANK',
-  COU = 'COU',
-  FORE = 'FORE',
-  GENT = 'GENT',
-  TRANS_LAND_TITLE = 'TRANS_LAND_TITLE',
-  REIV = 'REIV',
-  REPV = 'REPV',
-  SZL = 'SZL',
-  TAXS = 'TAXS',
-  VEST = 'VEST'
+  ABAN = 'TRANSFER DUE TO ABANDONMENT AND SALE',
```

Comment:
@doug-lovett I had to remove the codes and use the full description because of the Registration Type filter in MHR Table, otherwise after selecting the Type the code would be displayed in the field.

Reply:
Maybe the code could only be used when submitting the API request? Using the description is more brittle. If the match is not exact it won't work. The API returns the description when filtering by document type code.

Reply:
@doug-lovett I updated to use only the codes when filtering the table, but not all codes work. Maybe need to update API. I sent you a message in Teams.

Reply:
I added the mapping for codes, so the filtering is using the codes now, and it works. Thank you!